### ISSUE TITLE
Relative link (i.e. GitHub) support

### ISSUE THREAD

**Issue by vors** [tcort/markdown-link-check#2]
Hello, great project! Would like to use it for a [dotnet windows project](https://github.com/PowerShell/platyPS) with lots of docs in markdown.

I heavily use relative links and github is happy to resolve them.
Is there a way to do the same resolution with `markdown-link-check`?
Currently all relative links are considered broken.

**Comment by tcort** [comment]
I've implemented it and published a new version, version 1.2.0.

You can now use it like this:
```
$ ./markdown-link-check https://raw.githubusercontent.com/PowerShell/platyPS/master/docs/New-ExternalHelpCab.md
[✓] https://github.com/PowerShell/platyPS/blob/master/docs/New-ExternalHelpCab.md
[✓] New-ExternalHelp.md
[✓] New-MarkdownHelp.md
[✓] http://go.microsoft.com/fwlink/?LinkID=113216
```

**Comment by vors** [comment]
Cool! I also implemented my own module :)
https://github.com/vors/MarkdownLinkCheck
